packages/headless/src/interviews.ts

```diff
@@ -0,0 +1,100 @@
+import { useAtlas } from "@/atlas/index";
+import { IUser, IInterview, ICall, Paginated, Void } from "@litespace/types";
+import { useCallback } from "react";
+import { usePaginationQuery } from "@/query";
+import {
+  InfiniteData,
+  UseInfiniteQueryResult,
+  useMutation,
+  useQuery,
+} from "@tanstack/react-query";
+import { QueryKey } from "@/constants";
+
+type OnSuccess = Void;
+type OnError = (error: Error) => void;
+
+type useFindInterviewsProps = {
+  query: UseInfiniteQueryResult<
+    InfiniteData<
+      Paginated<{
+        interview: IInterview.Self;
+        call: ICall.Self;
+        members: ICall.PopuldatedMember[];
+      }>,
+      unknown
+    >,
+    Error
+  >;
+  list:
+    | {
+        interview: IInterview.Self;
+        call: ICall.Self;
+        members: ICall.PopuldatedMember[];
+      }[]
+    | null;
+  more: () => void;
+};
+
+export function useFindPagedInterviews(
+  profile: IUser.Self | null
+): useFindInterviewsProps {
+  const atlas = useAtlas();
+  const findInterviews = useCallback(
+    async ({ pageParam }: { pageParam: number }) => {
+      if (!profile) return { list: [], total: 0 };
+      return atlas.interview.findInterviews(profile.id, {
+        page: pageParam,
+        size: 10,
+      });
+    },
+    [profile]
+  );
+  return usePaginationQuery(findInterviews, [QueryKey.FindInterviewsPaged]);
+}
+
+export function useFindInterviews(profile: IUser.Self | null) {
```

Comment:
`useFindInterviews` is the same as `useFindPagedInterviews`. Please remove `useFindInterviews` and rename `useFindPagedInterviews` to `useFindInterviews`.

Reply:
@neuodev 
`useFindInterviews` is used in `TutorOnboarding` page as it expects an array of interviews, while usePaged is used in `interviews` page.

or would you like me to also change the logic in `TutorOnboarding` page to expect paged results?